mmcv/runner/optimizer/default_constructor.py

```diff
@@ -167,10 +177,18 @@ def add_params(self, params, module, prefix=''):
                         decay_mult = custom_keys[key].get('decay_mult', 1.)
                         param_group['weight_decay'] = self.base_wd * decay_mult
                     break
+
             if not is_custom:
-                # bias_lr_mult affects all bias parameters except for norm.bias
-                if name == 'bias' and not is_norm:
+                # bias_lr_mult affects all bias parameters
+                # except for norm.bias dcn.conv_offset.bias
+                if name == 'bias' and not (is_norm or is_dcn_submodule):
                     param_group['lr'] = self.base_lr * bias_lr_mult
+
+                if (prefix.find('offset') != -1 and is_dcn_submodule
```

Comment:
This may not be safe. How about a branch named xxx_offset in some model and it adopts a DCN?

Reply:
How about we also pass an argument to DCN's conv_offset like `reset_lr` or so? If `reset_lr=True` and the module is a DCN's submodule, the LR is reset by the `dcn_offset_lr_mult`.